client/layout/guided-tours/config-elements/link-quit.js

```diff
@@ -33,10 +33,10 @@ export default class LinkQuit extends Component {
 
 	render() {
 		const { children, primary, subtle, href, target } = this.props;
-		const classes = subtle ? 'guided-tours__subtle-button' : '';
+		const classes = subtle ? ' guided-tours__subtle-button' : '';
 		return (
 			<Button
-				className={ classes }
+				className={ 'guided-tours__button-link' + classes }
```

Comment:
Ah. Here it is: https://github.com/Automattic/wp-calypso/pull/11118#discussion_r99030059

I'd be fine with that being done in a follow-up PR though.